crates/turbopack-mdx/src/lib.rs

```diff
@@ -147,8 +152,48 @@ async fn into_ecmascript_module_asset(
         ..Default::default()
     };
     // TODO: upstream mdx currently bubbles error as string
-    let mdx_jsx_component =
-        compile(&file.content().to_str()?, &options).map_err(|e| anyhow!("{}", e))?;
+    let mdx_jsx_component = compile(&file.content().to_str()?, &options);
+
+    let mdx_jsx_component = match mdx_jsx_component {
+        Ok(mdx_jsx_component) => mdx_jsx_component,
+        Err(err) => {
+            let loc = Vc::cell(err.place.map(|p| {
+                let (start, end) = match *p {
+                    markdown::message::Place::Position(p) => (
+                        SourcePos {
+                            line: p.start.line,
```

Comment:
I know there is sometimes disagreement between 0 or 1 indexed, might be handy to have a comment here